test/functorch/test_eager_transforms.py

```diff
@@ -1877,13 +1877,20 @@ def test_chunk_jacrev(self, device, _preallocate_and_copy):
         def f(x, y):
             return (x.sin(), x + y), (x + 2, x.sum())
 
-        for chunk_size in [1, 2, 3, 4, 7, 10]:
+        for chunk_size in (1, 2, 3, 4, 7, 10, 1000):
             expected = jacrev(f, argnums=(0, 1))(x, y)
             actual = jacrev(f, argnums=(0, 1),
                             chunk_size=chunk_size,
                             _preallocate_and_copy=_preallocate_and_copy)(x, y)
             self.assertEqual(actual, expected)
 
+        err_msg = "jacrev: `chunk_size` should be greater than 0."
+        with self.assertRaisesRegex(ValueError, err_msg):
+            jacrev(f, argnums=(0, ), chunk_size=0)(x, y)
+
+        with self.assertRaisesRegex(ValueError, err_msg):
+            jacrev(f, argnums=(0, ), chunk_size=0)(x, y)
+
```

Comment:
Is the repitition intentional?

Reply:
Meant to have negative value for second. Thanks!